app/src/test/java/com/bitwarden/authenticator/data/authenticator/repository/AuthenticatorRepositoryTest.kt

```diff
@@ -89,12 +96,12 @@ class AuthenticatorRepositoryTest {
             fileManager = mockFileManager,
             importManager = mockImportManager,
             dispatcherManager = mockDispatcherManager,
+            settingRepository = settingsRepository,
         )
         assertEquals(
             SharedVerificationCodesState.FeatureNotEnabled,
             repository.sharedCodesStateFlow.value,
         )
-        verify(exactly = 0) { mockAuthenticatorBridgeManager.accountSyncStateFlow }
```

Comment:
We're subscribing to `accountSyncStateFlow` in `init` now, so this is no longer needed.

Subscribing to the flow with the feature flag off is OK, since we inject a no-op manager when the feature flag is off anyways.